pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -717,22 +717,15 @@ func (og *operationGenerator) GenerateUnmountDeviceFunc(
 	if err != nil {
 		return volumetypes.GeneratedOperations{}, deviceToDetach.GenerateErrorDetailed("UnmountDevice.NewDetacher failed", err)
 	}
-
-	volumeAttacher, err := attachableVolumePlugin.NewAttacher()
-	if err != nil {
-		return volumetypes.GeneratedOperations{}, deviceToDetach.GenerateErrorDetailed("UnmountDevice.NewAttacher failed", err)
-	}
-
 	unmountDeviceFunc := func() (error, error) {
-		deviceMountPath, err :=
-			volumeAttacher.GetDeviceMountPath(deviceToDetach.VolumeSpec)
+		deviceMountPath := deviceToDetach.DeviceMountPath
 		if err != nil {
 			// On failure, return error. Caller will log and retry.
 			return deviceToDetach.GenerateError("GetDeviceMountPath failed", err)
```

Comment:
Why does this line exists here? The `if` condition exists on an error that happened outside the `unmountDeviceFun`

Reply:
remove it. I forgot to clean it up. Thanks!